lms/templates/index.html

```diff
@@ -179,6 +179,83 @@ <h2 class="header-search-facets">${_('Refine Your Search')}</h2>
          if(getParameterByName('next')) {
```

Comment:
interesting https://stackoverflow.com/questions/4584373/difference-between-window-load-and-document-ready-functions